recipes/openfx/all/CMakeLists.txt

```
@@ -0,0 +1,72 @@
+cmake_minimum_required(VERSION 3.1)
+project(openfx VERSION 1.4.0 LANGUAGES CXX)
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+add_definitions(-D_HAS_AUTO_PTR_ETC)
+
+# Flags
+if(MSVC)
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DWINDOWS -DNOMINMAX")
+else()
+    set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra -Wno-deprecated -Wno-deprecated-declarations")
+endif()
+
+# Conan
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+# Macros
+set(SOURCE_SUBFOLDER ${CMAKE_CURRENT_SOURCE_DIR}/source_subfolder)
+
+# Sources
+set(OFX_HEADER_DIR "${SOURCE_SUBFOLDER}/include")
+set(OFX_SUPPORT_HEADER_DIR "${SOURCE_SUBFOLDER}/Support/include")
+set(OFX_SUPPORT_LIBRARY_DIR "${SOURCE_SUBFOLDER}/Support/Library")
+set(OFX_PLUGINS_HEADER_DIR "${SOURCE_SUBFOLDER}/Support/Plugins/include")
+set(OFX_HOSTSUPPORT_HEADER_DIR "${SOURCE_SUBFOLDER}/HostSupport/include")
+set(OFX_HOSTSUPPORT_LIBRARY_DIR "${SOURCE_SUBFOLDER}/HostSupport/src")
+
+file(GLOB_RECURSE OFX_HEADER_FILES "${OFX_HEADER_DIR}/*.h")
+file(GLOB_RECURSE OFX_SUPPORT_HEADER_FILES "${OFX_SUPPORT_HEADER_DIR}/*.h")
+file(GLOB_RECURSE OFX_SUPPORT_LIBRARY_FILES "${OFX_SUPPORT_LIBRARY_DIR}/*.cpp")
+file(GLOB_RECURSE OFX_PLUGINS_HEADER_FILES "${OFX_PLUGINS_HEADER_DIR}/*.H")
+file(GLOB_RECURSE OFX_HOSTSUPPORT_HEADER_FILES "${OFX_HOSTSUPPORT_HEADER_DIR}/*.h")
+file(GLOB_RECURSE OFX_HOSTSUPPORT_LIBRARY_FILES "${OFX_HOSTSUPPORT_LIBRARY_DIR}/*.cpp")
+
+# Support
+add_library(OfxSupport STATIC
+	${OFX_HEADER_FILES}
+	${OFX_SUPPORT_HEADER_FILES}
+	${OFX_SUPPORT_LIBRARY_FILES})
+
+target_include_directories(OfxSupport PUBLIC
+	${OFX_HEADER_DIR}
+	${OFX_SUPPORT_HEADER_DIR})
+
+# HostSupport
+add_library(OfxHost
+	${OFX_HEADER_FILES}
+	${OFX_HOSTSUPPORT_HEADER_FILES}
+	${OFX_HOSTSUPPORT_LIBRARY_FILES})
+
+target_link_libraries(OfxHost PUBLIC ${CONAN_LIBS})
```

```suggestion
target_link_libraries(OfxHost PUBLIC ${CONAN_LIBS})
```
Avoid both `CONAN_LIBS`and `CONAN_PKG::`, they will be deprecated on Conan 2.0. Instead, you can use `cmake_find_package` or `cmake_find_package_multi` and link directly to the target names (e.g. expat::expat)

---

yes, you need to add "cmake_find_package" generator at line 25 in you conanfile